benchmarks/common/include/benchmark/common/benchmark_getter.hpp

```diff
@@ -0,0 +1,53 @@
+/** Algebra plugins library, part of the ACTS project
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s)
+#include "benchmark_vector.hpp"
+
+namespace algebra::bench_op {
+
+// Functions to be benchmarked
+
+struct phi {
+  inline static const std::string name{"phi"};
+  template <typename vector_t>
+  decltype(auto) operator()(const vector_t &a) const {
```

Comment:
I still have a deep distrust for `decltype(auto)`. We could just use `auto` here as well, no?

Reply:
Sure